src/main/java/htsjdk/samtools/cram/io/ExternalCompression.java

```diff
@@ -149,4 +150,25 @@
         buffer.get(bytes);
         return bytes;
     }
+
+    public static byte[] uncompress(final BlockCompressionMethod method, final byte[] compressedContent) {
+        try {
+            switch (method) {
+                case RAW:
+                    return compressedContent;
+                case GZIP:
```

Comment:
There's an opportunity for refactoring the compression classes here.  Leaving alone for this PR.